tests/utils/payloads.py

```
@@ -163,23 +165,26 @@ def response_handler(self, response: Any) -> str:
     def validate(self, response: Any, content: str) -> None:
-        pattern = r'dynamo_component_requests_total\{[^}]*model="[^"]*"[^}]*\}\s+(\d+)'
+        requests_total_name = prometheus_names.work_handler.requests_total
+        pattern = (
+            rf'{re.escape(requests_total_name)}\{{[^}}]*model="[^"]*"[^}}]*\}}\s+(\d+)'
+        )
         matches = re.findall(pattern, content)
         if not matches:
             raise AssertionError(
-                "Metric 'dynamo_component_requests_total' with model label not found in metrics output"
+                f"Metric '{requests_total_name}' with model label not found in metrics output"
             )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)